### PR TITLE
Implement progress views for iOS (GSoC'23)

### PR DESCRIPTION
**Pull request by dhruvshrivastava18**
Added the Circular progress and Linear progress for iOS.

### CODE REVIEW

**Comment by AppInventorWorkerBee** [comment]
Can one of the admins verify this patch?

File: appinventor/components-ios/src/CircularProgress.swift [hunk 1]
```diff
@@ -0,0 +1,101 @@
+// -*- mode: swift; swift-mode:basic-offset: 2; -*-
+// Copyright 2017-2023 MIT, All rights reserved
```
Comment (ewpatton) [inline appinventor/components-ios/src/CircularProgress.swift:2]: Since this is new to App Inventor this year, you only have to use 2023.

**Comment by ewpatton** [comment]
@AppInventorWorkerBee ok to test

File: appinventor/components-ios/src/CircularProgress.swift (hunk 1 above)
Comment (dhruvshrivastava18) [reply appinventor/components-ios/src/CircularProgress.swift]: @ewpatton updated to 2023

**Comment by ewpatton** [comment]
@barreeeiroo Have you identified someone on the Kodular side to port the Android version over to AI2?

**Comment by barreeeiroo** [comment]
> @barreeeiroo Have you identified someone on the Kodular side to port the Android version over to AI2?

Yes, I'll be migrating them once the iOS version is finished and ready to ship.

**Review by ewpatton — APPROVED** [review]

**Pull request merged**